app/grandchallenge/core/templates/challenge.html

```diff
@@ -33,79 +33,78 @@
     {{ block.super }}
 {% endblock %}
 
-{% block sidebar %}
+{% block topbar %}
     {% if challenge %}
-        <div class="col-12 col-md-3 col-lg-2 mb-3">
-            <ul class="nav nav-pills flex-column">
-                {% for page in pages %}
-                    {% if not page.hidden %}
-                        <li class="nav-item">
-                            <a class="nav-link {% if page == currentpage %}active{% endif %}"
-                               href="{{ page.get_absolute_url }}">
-                                {% filter title %}
-                                    {% firstof page.display_title page.title %}
-                                {% endfilter %}
-                            </a>
-                        </li>
-                    {% endif %}
-                {% endfor %}
-
-                {% if challenge.display_forum_link %}
+        <div class="row">
+            <div class="col-md-10">
+                <div class="nav nav-tabs" role="tablist">
                     <li class="nav-item">
-                        <a class="nav-link"
-                           href="{% url 'forum:forum' slug=challenge.forum.slug pk=challenge.forum.pk %}">
-                            Forum
+                        <a class="nav-link {% if request.resolver_match.view_name == 'pages:detail' %} show active{% endif %}"
```

Comment:
Does this view exist? I think not, maybe add a TODO to fix before it goes on master.

Reply:
well, normally the pages detail view would require `page_title` as slug to work, but it seems to work just fine without the slug here (no resolver error or anything), not sure why.